torchvision/datapoints/_bounding_box.py

```diff
@@ -10,12 +10,35 @@
 
 
 class BoundingBoxFormat(StrEnum):
+    """[BETA] Coordinate format of a bounding box.
+
+    Available formats are
+
+    * ``XYXY``
+    * ``XYWH``
+    * ``CXCYWH``
+    """
+
     XYXY = StrEnum.auto()
     XYWH = StrEnum.auto()
     CXCYWH = StrEnum.auto()
 
 
 class BoundingBox(Datapoint):
+    """[BETA] :class:`torch.Tensor` subclass for bounding boxes.
+
+    Args:
+        data: Any data that can be turned into a tensor with :func:`torch.as_tensor`.
```

Comment:
No strong opinion, numpy and scikit-learn use `array-like` a lot I believe

```suggestion
        data (tensor-like): Any data that can be turned into a tensor with :func:`torch.as_tensor`.
```